### ISSUE TITLE
Some generated files are not well-formated

### ISSUE THREAD

**Issue by lichuqiang** [kubernetes-sigs/kubebuilder#149]
When walking through the files generated through `kubebuilder`, I found codes in some files are not well-aligned, for example in `cmd/controller-manager/main.go`:
```
import (
        "flag"
        "log"
    
    // Import auth/gcp to connect to GKE clusters remotely
    _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
    
    configlib "github.com/kubernetes-sigs/kubebuilder/pkg/config"
        "github.com/kubernetes-sigs/kubebuilder/pkg/inject/run"
        "github.com/kubernetes-sigs/kubebuilder/pkg/install"
    "github.com/kubernetes-sigs/kubebuilder/pkg/signals"
        extensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
    
    "example.com/pkg/inject"
    "example.com/pkg/inject/args"
)

var installCRDs = flag.Bool("install-crds", true, "install the CRDs used by the controller as part of startup")

// Controller-manager main.
func main() {
        flag.Parse()

    stopCh := signals.SetupSignalHandler()
```

Of course a user can easily format the files via `go fmt`, but for better experience, I think we'd better
1. modify existing templates and the place to generate files, to ensure the generated codes are well-formated
2. or maybe we can exec `go format` for output files automatically

@pwittrock WDYT?

**Comment by Liujingfang1** [comment]
@lichuqiang I found a recent comment #147 exec `go format` on the generated codes.

**Comment by lichuqiang** [comment]
@Liujingfang1 Thanks!
Closing the issue since it's already fixed by #147
